nixos/modules/testing/test-instrumentation.nix

```diff
@@ -89,6 +89,21 @@ in
 
     sshBackdoor = {
       enable = mkEnableOption "vsock-based ssh backdoor for the VM";
+      vsockOffset = mkOption {
```

Comment:
hm we somehow got this duplicated. not sure if necessary?

Reply:
This is a different level: the definition in nodes.nix is for the module-system options on driver-level, this is at NixOS machine-config level.

That said, I don't see why this is necessary anymore. I'll simplify this.

Reply:
Done!